sketchlib/Tween.js

```diff
@@ -0,0 +1,76 @@
+import { Point } from "../pga2d/objects.js";
+import { clamp } from "./clamp.js";
+
+function lerp_number(a, b, t) {
+  return (1 - t) * a + t * b;
+}
+
+/**
+ * @template T
+ */
+export class Tween {
+  /**
+   * Create a tween to interpolate between the start and end value. Right now
+   * this only supports linear interpolation
```

Comment:
Finish this documentation comment. Also add a note that this constructor should not be used directly, see `Tween.point` and `Tween.scalar`